MeshLib/Properties.h

```diff
@@ -153,21 +150,40 @@ class Properties
 						boost::any_cast<PropertyVector<T> const&>(it->second)
 					);
 			} catch (boost::bad_any_cast const&) {
-				ERR("A property with the specified data type is not available.");
 				return boost::optional<PropertyVector<T> const&>();
 			}
 		} else {
-			ERR("A property with the specified name and MeshItemType is not available.");
+			ERR("A property with the specified name is not available.");
 			return boost::optional<PropertyVector<T> const&>();
 		}
 	}
 
-	void removeProperty(std::string const& name,
-		MeshItemType mesh_item_type)
+	/// Method to get a vector of property values.
+	template <typename T>
+	boost::optional<PropertyVector<T>&>
+	getProperty(std::string const& name)
```

Comment:
Yes. Actually there is a const version of `getProperty` in [line 142](https://github.com/ufz/ogs/pull/607/files#diff-91ee186ac317eab2fe2b6458c84f37bdR142). This non-const version is used for writing back the results etc.